README.md

```diff
@@ -154,7 +153,7 @@ in addition to the table-specific fields listed below.
     * `diff_length`: Length (in characters) of the diff identified by the above `diff_with_previous_url`.
     * `diff_hash`: SHA 256 hash of the above diff identified by `diff_with_previous_url`.
     * `diff_with_first_url`: URL to diff view in Versionista (comparing with the first recorded version)
-    * `has_content`: Boolean indicating whether Versionista had raw content for this version. If this is true, the version’s `uri` should have a value (and vice-versa).
+    * `has_content`: Boolean indicating whether Versionista had raw content for this version. If this is true, the version’s `Uri` should have a value (and vice-versa).
```

Comment:
Same as above: `uri` is actually the name of a field, so it’s important that it’s all lower-case.